src/psyclone/tests/dynamo0p3_transformations_test.py

```diff
@@ -607,23 +660,20 @@ def test_colouring_after_openmp():
     '''Test that we raise an error if the user attempts to colour a loop
     that is already within an OpenMP parallel region. We test when
     distributed memory is on or off '''
-    # For this test we must use a kernel that doesn't actually require
-    # colouring as otherwise PSyclone won't let us apply the OpenMP
-    # transformation first!
     _, info = parse(os.path.join(os.path.dirname(os.path.abspath(__file__)),
                                  "test_files", "dynamo0p3",
-                                 "9_orientation.f90"),
+                                 "1_single_invoke.f90"),
```

Comment:
This is because 9_orientation calls the testkern_orientation kernel which writes to a discontinuous field. In this PR we have disabled support for colouring discontinuous loops so this test would no longer work. I could have changed to testkern_orientation2 (which is done in the previous loop) but testkern in 1_single_invoke is a simpler kernel so I chose that.